### PR TITLE
DX tweaks

### PR DESCRIPTION
**Pull request by weshooper**
1. Updated dependencies and applied latest CS fixer
2. Added `dependabot.yml`
3. Simplified `semaphore.yml`:
	- Caching CS fixer output takes longer than running it for such a small project
	- We can rely on `cache` command defaults for keys
	- We don't need to worry about `execution_time_limit`
	- Simpler (and faster) to use one block/task for such a small project
4. Added [ergebnis/composer-normalize](https://github.com/ergebnis/composer-normalize) (link explains the benefits)
5. Added two Composer scripts/aliases
6. Tidied up the README

Feedback from @fusionspim/devs on the relevancy of the 'Future plans' section appreciated 😄

### CODE REVIEW

File: .semaphore/semaphore.yml [hunk 1]
```diff
@@ -5,34 +5,19 @@ agent:
...
-      jobs:
-      - name: Tests
-        commands:
-          - cache restore php-cs-fixer-cache
-          - php-cs-fixer fix --config=.php_cs.dist --dry-run --diff -v --stop-on-violation
-          - cache store php-cs-fixer-cache .php_cs.cache
-          - phpunit --stop-on-failure
+        - name: CS & unit tests
+          commands:
+            - phpbrew use php-7.4.6
```
Comment (maxakropolis) [inline .semaphore/semaphore.yml:15]: There might be a problem with this now - see https://github.com/fusionspim/george-app/pull/846/files#diff-ff188fb6aae9d0e400ce7963e3543cf3R1

File: README.md [hunk 2]
```diff
@@ -1,27 +1,23 @@
...
-## Install
-Via Composer:
+**Parsing**: Retrieve category, assignment, product, variation, set and bundle information from an XML file, either as an array, or yield using generators. Maps IDs to the more useful elements and, optionally, attributes.
 
-``` bash
-$ composer require fusionspim/php-demandware-xml
-```
+## Installation
 
+Run `composer require --dev fusionspim/php-demandware-xml`.
```
Comment (maxakropolis) [inline README.md:13]: Why `--dev` ?

File: .semaphore/semaphore.yml (hunk 1 above)
Comment (weshooper) [reply .semaphore/semaphore.yml]: Fixed, ta.

File: README.md (hunk 2 above)
Comment (weshooper) [reply README.md]: No idea! Was like that before and didn't spot it. Have removed ` --dev` ta.

**Pull request merged**